main.js

```diff
@@ -0,0 +1,35 @@
+var board = {
+  pegs: [
+    [5, 4, 3, 2, 1],
+    [],
+    []
+  ],
+  moveDisc: function (startPeg, endPeg) {
+    startPeg -= 1;
+    endPeg -= 1;
+    if (this.pegs[endPeg] > 0 && this.pegs[endPeg] < this.pegs[startPeg]){
+      console.log(`You cannot move a larger disc on top of a smaller one, board is still: `)
+    } else {
+      startPeg = this.pegs[startPeg].pop();
+      endPeg = this.pegs[endPeg].push(startPeg);
+      printBoard;
+      this.checkWinner();
+    } 
+    var printBoard = this.pegs.map(function (peg){
```

Comment:
Remember that map returns a new array which means that you need to `return` from this function.